elements/lisk-p2p/src/peer/base.ts

```diff
@@ -30,16 +51,26 @@ import {
 	ProtocolMessagePacket,
 	ProtocolNodeInfo,
 } from '../p2p_types';
-import { constructPeerIdFromPeerInfo, getNetgroup } from '../utils';
-
-import * as socketClusterClient from 'socketcluster-client';
-import { SCServerSocket } from 'socketcluster-server';
 import {
+	constructPeerIdFromPeerInfo,
+	getNetgroup,
 	validatePeerInfo,
 	validatePeersInfoList,
 	validateProtocolMessage,
 	validateRPCRequest,
-} from '../validation';
+} from '../utils';
+
+type SCClientSocket = socketClusterClient.SCClientSocket;
+
+// Can be used to convert a rate which is based on the rateCalculationInterval into a per-second rate.
+const RATE_NORMALIZATION_FACTOR = 1000;
```

Comment:
should this also be in `constants`?

Reply:
I've kept some local constants in their files depending on the scope. For example this one wasn't being exported, so for now we should keep it in its enclosed scope. Do you foresee any use in another scope?